modules/ROOT/pages/view-partner-details.adoc

```diff
@@ -0,0 +1,14 @@
+= View a Partner's Details
```

Comment:
```suggestion
= View and Update Partner Details
```

Reply:
But how is this different from Update Partner Settings?

Reply:
I meant to remove this topic.